en/docs/build-and-run/build-and-run.md

```diff
@@ -1,69 +1,78 @@
-# Build and run the program
+# Build and run a Ballerina program
 
-Follow the steps below to create a sample Ballerina program in VS Code. 
+You can build your Ballerina program and run it either by using the **Code** view or the **Diagram** view when you use the Ballerina extension.
 
-## Use the Code View
+## Use the Code view
 
->**Note:** Skip 1-3 steps if you already have a Ballerina code.
+Let's build and run a sample Ballerina program using the Code view.
```

Comment:
Shall we move the `run a sample Ballerina program` part (steps 1 to 3) under Get Started and say `Try your sample` or `Try your first Ballerina program`?